pkg/client/introduce/client.go

```diff
@@ -1,226 +0,0 @@
-/*
```

Comment:
Don't we need this file ?

Reply:
I see it coming back in the next PR.

Reply:
I cannot use the previous client. The service was changed. Logic now is different. New The client is implemented e in this PR  https://github.com/hyperledger/aries-framework-go/pull/1254